### DISCUSSION TITLE
Is there any reason for Model turrets and guns being so heavy?

### DISCUSSION THREAD

**Discussion by junetried** [endless-sky/endless-sky#9848] [General]
This is admittedly a very minor thing, but I would imagine having a model of a turret or gun means that a significant portion of the weight (the mass) doesn't exist, which would make it much lighter than the outfit space it uses would suggest. Despite that, the model weapons have full mass. Are they weighted? That seems expensive and unnecessary, even if these are collector's items. Am I overthinking it?

**Comment by bene-dictator** [comment]
To some extent the "rule of cool". The Plasma Turret just looks badass (thanks Becca). To another extent I don't think it has really been thought about. Some weapons do incorporate the idea that their mass is separate from weapons space (whether intentionally I don't know), like the Heavy Ion Cyclotron, and I think some newer outfits like the Tractor Beam, the Gatling Turret, and the Javelin Turret.

**Reply by junetried** [reply]
Oh, sorry if I wasn't clear enough: I'm talking about the model Coalition weapons that are for sale to civilians.

**Reply by bene-dictator** [reply]
The MCS Extractor?

**Reply by junetried** [reply]
Forgive me, I thought I had disabled my plugins. I think it's something from a plugin.

**Reply by bene-dictator** [reply]
No problem. I guess you mean the Celebration Module from Zoura's Outfits Expanded, maybe I could ask Zoura about that directly and get it changed.

**Reply by junetried** [reply]
I was curious so I went through my plugins to check. I was talking about [this outfit](https://github.com/MidnightPlugins/Lost-in-Midnight/blob/184059cad48793ccc01bcc1fae4e50c6500c977d/data/coalition%20midnight/Outfits%20Coalition.txt#L143) from [Lost-in-Midnight](https://github.com/MidnightPlugins/Lost-in-Midnight/tree/main).
